### PR TITLE
Create email list page for authflow v2 (PR v2)

### PR DESCRIPTION
**Pull request by Kuixz**
On top of nothing

### CODE REVIEW

File: resources/authgear/templates/en/web/authflowv2/settings_identity_list_email.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "BackTitle" (translate "v2.component.navbar.default.item-back-button-label" nil)
+        "BackHref" (call $.MakeURL "/settings")
+        "Title" (translate "v2.page.settings-identity-list-email.default.navbar-title" nil)
+     )
+  }}
+{{ end }}
+
+{{ define "page-content" }}
+emails
+{{ end }}
```
Comment (cychiuae) [inline resources/authgear/templates/en/web/authflowv2/settings_identity_list_email.html]: Please add a newline at the end of this file

File: resources/authgear/templates/en/translation.json [hunk 2]
```diff
@@ -1114,6 +1114,7 @@
   "v2.page.select-account.default.title": "Log in to {AppOrClientName}",
   "v2.page.select-account.default.use-another-account": "Use another account",
   "v2.page.settings-change-password.default.navbar-title": "Change Password",
+  "v2.page.settings-identity-list-email.default.navbar-title": "Email",
```
Comment (cychiuae) [inline resources/authgear/templates/en/translation.json]: Please also add translation in zh-hk and zh-tw. 

en, zh-hk and zh-tw need us to manually add the translations

**Comment by cychiuae** [comment]
1. Please remove the commits by york. This PR doesn't use any of those.
2. Resolve the merge conflicts
3. Add back the translations for zhhk and zhtw
4. Add the newline character at the end of `[resources/authgear/templates/en/web/authflowv2/settings_identity_list_email.html](https://github.com/cychiuae/authgear-server/pull/27/files/f71d8ae27602c8e1d8817bafc2e01dd93f4e8a26..e815423be8262c0a1bee61e53f273fb93d6a506a#diff-f1f9a5861bc9d5cfc82a31e2d7b50bb9113e8f1189c4fe4a73b7581a9efb0cd0)`

Then this PR is okay to merge

**Comment by Kuixz** [comment]
1. Completed and edited description
2. Completed but kind of funny
3. Completed but mtl
4. Completed

@cychiuae

**Comment by cychiuae** [comment]
@Kuixz Please remove `pkg/auth/handler/webapp/authflowv2/settings_identity_view_email.go` from commit [340e304](https://github.com/cychiuae/authgear-server/pull/27/commits/340e3048ca54b10a3c49d98a5897b6aa0589917f)

**Comment by cychiuae** [comment]
@Kuixz I pushed 2 commits to your branch

**Comment by Kuixz** [comment]
Seems missing primary email section. Will add it in primary email PR.

**Pull request merged**